src/Compilers/Core/Portable/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.cs]
+dotnet_diagnostic.RS0041.severity = none
```

Comment:
Do you want a tracking issue to remove this?

Reply:
We don't need to add a blanket suppression. A few files need a pragma. See [https://github.com/dotnet/roslyn/pull/41000](https://github.com/dotnet/roslyn/pull/41000)

---
In reply to: [565227031](https://github.com/dotnet/roslyn/pull/50793#discussion_r565227031) [](ancestors = 565227031)